modules/index-service/src/main/java/org/opencastproject/index/service/resources/list/api/ResourceListProvider.java

```diff
@@ -56,5 +56,14 @@
    * @param listName the name of the list
    * @return if the results should be translated
    */
-  boolean isTranslatable(String listName);
+  public abstract boolean isTranslatable(String listName);
+
+  /**
+   * Returns the key of the default value in the list if customized, else null.
+   *
+   * @return key of default value or null
+   */
+  public String getDefault() {
```

Comment:
As follow-up to the previous comments of @karendolan:

Personally, I share the preference of @karendolan to don't hide this from the implementation, but there are arguments pro and contra that.

What should not be the case, however, is that different functions of the smae interface handle this decision differently: The same argument would hold for the method isTranslatable(), but in that case, each implementation does implement the method (even if a default would be "less lines" in overall).

Maybe quickly talk to @lkiesow (IIRC, he added isTranslatable) about his opinion.

Reply:
@wsmirnow  wrote that. I have no strong opinion on this btw.

Reply:
@staubesv I do see your point regarding the inconsistency between getDefault() and getTranslatable(). Since the majority seems to favor it, I will change getDefault() so that it's implemented in each provider.